__tests__/chat.test.ts

```diff
@@ -77,7 +76,7 @@ beforeAll(async () => {
     await initCandidatePage(candidatePage);
 });
 
-test("An employer can create a new chat");
+// test("An employer can create a new chat");
```

Comment:
a test without callback is already skipped (but seen as yellow) so revert this change

Reply:
I think you miss this one